server.js

```diff
@@ -56,6 +56,25 @@ app.get('/repos/:user', async (req, res) => {
 		.send(await request(`https://img.shields.io/badge/Repos-${response.public_repos}-brightgreen${req.originalUrl.slice(req.originalUrl.indexOf('?'))}`).raw())
 })
 
+app.get('/gists/:user', async (req, res) => {
+	const { user } = req.params;
+
+	const response = await request(
+		`https://api.github.com/users/${user}/gists`
+	).header(githubHeaders).json();
+
+	if (response.length === 0)
```

Comment:
if you search for a user that does not exist, github returns this
```
{
    "message": "Not Found",
    "documentation_url": "https://docs.github.com/rest/reference/gists#list-gists-for-a-user"
}
```

checking for length only is not the way to go i guess